### ISSUE TITLE
pulumi new fails when stack name is specified via --stack and logged into the local backend

### ISSUE THREAD

**Issue by justinvp** [pulumi/pulumi#3078]
```
$ pulumi new typescript --stack foo
error: failed to load checkpoint: blob (code=NotFound): stat /Users/justin/.pulumi/stacks/foo.json: no such file or directory
```

This is because it checks to see if the stack exists to support pulling down already created stacks from https://app.pulumi.com/site/new-project. But the `--stack` is still useful as a way of specifying the stack name to avoid being prompted for it. To avoid this error for the local case, `pulumi new` could just avoid doing the stack exists check when not logged into the cloud backend.

**Comment by snowsky** [comment]
@justinvp I think you are right and I linked the other issue. Will take a look during the weekend.

**Comment by justinvp** [comment]
@snowsky, I believe this was actually already fixed with #3131, so I'm going to close this issue. Though, the experience today isn't as good as it could be due to #3077.
